cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -95,12 +95,18 @@ object ReporterCommand : CommandWithHelp() {
         resolutionsFile?.readValue<Resolutions>()?.let { resolutionProvider.add(it) }
 
         reportFormats.distinct().forEach {
+            val name = it.toString().substringBefore("Reporter")
             try {
-                it.generateReport(ortResult, resolutionProvider, outputDir)
+                val reportFile = it.generateReport(ortResult, resolutionProvider, outputDir)
+                if (reportFile != null) {
+                    println("Created '$name' report:\n\t$reportFile")
+                } else {
+                    println("Could not create '$name' report.")
```

Comment:
"Could not" sounds like an error, see also below, whereas in case of a `NOTICE` file writing no output is not necessarily caused by an error. Maybe better: "Not creating any output for '$name' report."

Reply:
Done.

Reply:
Done.